autoexec.bat.template

```diff
@@ -4,7 +4,8 @@ cd c:\flash
 cls
 echo.
 echo.
-echo This will flash your firmware to support a classic 7-row keyboard
+echo This will flash your EC firmware to support a classic 7-row keyboard (default)
+echo or other modifications as configured at modification time.

```

Comment:
The build system should probably keep track of which patches are enabled and applied and insert that here, to be 100% clear and correct